qmail-popup.c

```diff
@@ -88,7 +88,6 @@ char *pass;
   int wstat;
   int pi[2];
  
-  if (fd_copy(2,1) == -1) die_pipe();
```

Comment:
I think I understand why you did this here, but will you explain it so I'm sure?  If it's necessary the qmail-popup.8 manpage needs to be corrected, as it explains how these file descriptors are used.

Reply:
Totally missed that in the manpage. I did this proximately because pop3d couldn't write a log entry by writing to stderr, and ultimately because I've never understood why the interface was defined that way. [qmail.org's Tips & Advice](http://qmailorg.schmonz.com/top.html#tips) suggests this nutty workaround:

> Qmail-popup redirects stderr to stdout, thus making it impossible to write a wrapper around qmail-pop3d which writes to the logfile by writing to stderr. Being a little cleverer with the shell, you can also redirect FD 7 onto stdout like this:
> 
> `/var/qmail/bin/qmail-pop3d-wrapper.sh /var/qmail/bin/qmail-pop3d Maildir 2>&1 7>&1`
> 
> Once you've done that, qmail-pop3d-wrapper.sh can log to FD 7, like this:
> 
> `#!/bin/sh`
> `echo "qmail-pop3d: user $USER logged in from $TCPREMOTEIP:$TCPREMOTEPORT" >&7`
> `$@`

So I changed the popup interface to make it possible to write a log entry the same way everything else in qmail does. If we agree that on that change, I'll update the popup manpage to match.

Reply:
> I changed the popup interface to make it possible to write a log entry the same way everything else in qmail does. If we agree that on that change, I'll update the popup manpage to match.

It may not actually, literally be "everything else," but your point is well taken--the exceptions if they exist probably support your point rather than contradict it.

Would you make this work a separate commit in this PR?  I'll need to do a run through of the code for this behavior elsewhere, for due diligence, which may scare up other material to record, change, or accommodate, but on it's own merit this change is reasonable and superior to the workaround.